routing/routing.go

```diff
@@ -68,6 +68,7 @@ var ApiRoutes = []APIRoute{
 	{"authMethod:ListOne", "GET", "/service-types/{service-type}/hosts/{host}/authM", handlers.AuthMethodListOne},
 	{"bindings:ListAllByServiceTypeAndHost", "GET", "/service-types/{service-type}/hosts/{host}/bindings", handlers.BindingListAllByServiceTypeAndHost},
 	{"bindings:ListOneByDN", "GET", "/service-types/{service-type}/hosts/{host}/bindings/{dn}", handlers.BindingListOneByDN},
+	{"bindings:update", "PATCH", "/service-types/{service-type}/hosts/{host}/bindings/{dn}", handlers.BindingUpdate},
```

Comment:
use put not patch

Reply:
Changed PATCH to PUT. Swagger and docs updated aswell